py-setup

```diff
@@ -7,14 +7,25 @@ set -x
 printf "travis_fold:start:py-setup\n"
 
 TARGET=${TARGET:-..}
-
+EXCLUDE=${EXCLUDE:-}
+DIR=${DIR:-.}
 dir=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
 cd $TARGET
 pip install --upgrade pip setuptools
 pip install -r $dir/requirements.txt
-flake8 -v .
+array=( $EXCLUDE )
```

Comment:
This makes me wonder if we don't want linting in a script. And _that_ makes me wonder if we don't want a default file (including `release` and `version-bump`) in this repo and it gets used IFF `.omeroci` doesn't provide one.

Reply:
removing from check from ``py-setup`` will be necessary since I am now having to combine 2 "dockers" => flake8 is run twice